pkgs/applications/blockchains/alfis/default.nix

```diff
@@ -23,7 +23,12 @@ rustPlatform.buildRustPackage rec {
     sha256 = "sha256-E0n1keNk5jNnErNvYhb8oe26kK9Opl+IJ5zpsvrqS84=";
   };
 
-  cargoSha256 = "sha256-kbo3OMLYA/5xctz/YhQNd8IYlyCQB7D/8rCHZwjvlMI=";
+  # https://github.com/Boscop/web-view/issues/292
+  cargoPatches = [
+    ./webview-sys.patch
```

Comment:
This change should probably be submitted upstream and `fetchpatch`'ed here.

Reply:
Opened PR at Revertron/Alfis#320.

Reply:
Upstreamed.